src/app/myworkflows/my-workflow/my-workflow.component.html

```diff
@@ -54,6 +54,20 @@
                         >
                           <mat-icon>add</mat-icon>
                         </button>
+                        <button mat-icon-button [matMenuTriggerFor]="myWorkflowsActions"><mat-icon>more_vert</mat-icon></button>
+                        <mat-menu #myWorkflowsActions="matMenu">
+                          <button
+                            mat-menu-item
+                            color="primary"
+                            type="button"
+                            (click)="addToExistingWorkflows()"
+                            [disabled]="isRefreshing$ | async"
+                            matTooltip="Discover workflows added by others using your linked source control accounts"
```

Comment:
I think an explicit question mark button would be better. I've found that many users don't know to look for tooltips.

That could wait for a future PR, as it's a bit more work to add.